Doc/library/xmlrpc.rst

```diff
@@ -1,3 +1,5 @@
+.. module:: xmlrpc
```

Comment:
Please could you add this under the header line and with a `:synopsis:` like in other library files?